.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: node_js
+node_js:
+  - "5"
+before_script:
+  - npm install grunt-cli -g
+env:
+  - CXX=g++-4.8  
+addons:
+  apt:
+    sources:
+      - ubuntu-toolchain-r-test
+    packages:
+      - gcc-4.8
+      - g++-4.8
```

Comment:
I got the g++-4.8 from both the DNSChain file and a solution to the travis-node build issue I found via googling. The gcc-4.8 isn't in the DNSChain file though: https://github.com/okTurtles/dnschain/blob/master/.travis.yml